modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+
+.Enabling Assisted Installer Service on Bare Metal
```

Comment:
Wanted to verify if this is the final name?

Reply:
I believe so, but I'll follow up.

Reply:
Updated as suggested.